node/lib/util/status_util.js

```diff
@@ -613,3 +613,36 @@ Before proceeding, you must complete the ${command} in progress (by running
 'git meta ${command} --abort').`);
     }
 };
+
+/**
+ * Return an error message if the specified `status` of a repository isn't in a
+ * normal, ready state, that is, it does not have any conflicts or in-progress
```

Comment:
This is hard to parse.  There are too many negatives.  Can we rephrase?

Reply:
interesting, is this a github bug, because I don't think I made any changes to [this file](https://github.com/twosigma/git-meta/blob/master/node/lib/util/status_util.js#L585-L592)